build/Targets/Analyzers.Versions.targets

```diff
@@ -1,18 +1,18 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <PropertyGroup>
-      <MetaCompilationSemanticVersion>1.1.0</MetaCompilationSemanticVersion>
+      <MetaCompilationSemanticVersion>1.2.0</MetaCompilationSemanticVersion>
       <MetaCompilationPreReleaseVersion>beta1</MetaCompilationPreReleaseVersion>
 
-      <MicrosoftCodeAnalysisAnalyzersSemanticVersion>1.1.0</MicrosoftCodeAnalysisAnalyzersSemanticVersion>
+      <MicrosoftCodeAnalysisAnalyzersSemanticVersion>1.2.0</MicrosoftCodeAnalysisAnalyzersSemanticVersion>
       <MicrosoftCodeAnalysisAnalyzersPreReleaseVersion>beta1</MicrosoftCodeAnalysisAnalyzersPreReleaseVersion>
 
-      <SharedLibrarySemanticVersion>1.1.0</SharedLibrarySemanticVersion>
+      <SharedLibrarySemanticVersion>1.2.0</SharedLibrarySemanticVersion>
       <SharedLibraryPreReleaseVersion>beta1</SharedLibraryPreReleaseVersion>
 
-      <SharedTestSemanticVersion>1.1.0</SharedTestSemanticVersion>
+      <SharedTestSemanticVersion>1.2.0</SharedTestSemanticVersion>
       <SharedTestPreReleaseVersion>beta1</SharedTestPreReleaseVersion>
 
-      <RoslynDiagnosticsAnalyzersSemanticVersion>1.1.1</RoslynDiagnosticsAnalyzersSemanticVersion>
+      <RoslynDiagnosticsAnalyzersSemanticVersion>1.2.0</RoslynDiagnosticsAnalyzersSemanticVersion>
```

Comment:
It was so even before my recent change - I presume it got bumped up to 1.1.0 some time back and probably needed a fix to change to 1.1.1 - I kept this unchanged when I recently moved all other packages from 1.0.0 to 1.1.0